_posts/2020-02-29-f12.markdown

```diff
@@ -33,11 +33,11 @@ one would write the following shell commands:
 {% highlight shell %}
 # Create an empty FAT12 formatted image
 mkfs.msdos -C test.flp 1440
-# Write a bootloader to it
+# Write a bootloader for it
```

Comment:
```suggestion
# Write a bootloader to it
```

Hier möchte ich sagen, dass ich den Bootlaoder **auf** das Image schreiben will

Reply:
Gut, dann: Write the bootloader for it